src/Run.hs

```diff
@@ -69,8 +71,14 @@ run dir startupFile args = do
       go = do
         status <- withSession startupFile args $ \ session -> do
           let
-            triggerAction = saveOutput (trigger session)
-            triggerAllAction = saveOutput (triggerAll session)
+            mkTriggerAction theTrigger = do
+              result@(success, _) <- theTrigger session
+              saveOutput (return result)
```

Comment:
I think this changes what `seito` will do when a test run is in progress:

Before: `seito` will wait until reload / test finished and use recent results.
Now: `seito` will get stale results.

Reply:
I think handling hooks in `saveOutput` instead could turn out nice.

Reply:
done